frontend/globals/unity-util.ts

```diff
@@ -1415,6 +1415,28 @@ export class UnityUtil {
 		UnityUtil.toUnity('SetUnits', UnityUtil.LoadingState.MODEL_LOADING, units);
 	}
 
+	/**
+	 * Enable orthographic view
+	 */
+	public static enableOrthographic() {
+		UnityUtil.toUnity('EnableOrthographic', UnityUtil.LoadingState.MODEL_LOADING, undefined);
+	}
+
+	/**
+	 * Disable orthographic view. Enables perspective view.
+	 */
+	public static disableOrthographic() {
+		UnityUtil.toUnity('DisableOrthographic', UnityUtil.LoadingState.MODEL_LOADING, undefined);
+	}
+
+	/**
+	 * Set orthographic size for orthographic view
+	 * @param {number} orthographicSize - Camera's half-size when in orthographic mode
+	 */
+	public static setOrthographicSize(orthographicSize: number) {
```

Comment:
I don't think this should be exposed, unless we have a UI for the user to tweak this